packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -263,12 +263,12 @@ $ms-Persona-presenceSizeXl: 28px;
 
 //== Modifier: Tiny Persona
 //
-.root.rootIsTiny {
-  height: $ms-Persona-sizeTiny;
-  min-width: $ms-Persona-sizeTiny;
+.root.rootIs12 {
```

Comment:
Can you rename all of these to rootIsSizeXX, for more descriptive and consistent class names?